README.md

```diff
@@ -46,6 +46,15 @@ Pass the path to configuration files on the command line with `-c` or `--config-
 If multiple configuration files are provided, they will be merged together.
 If a configuration key occurs in more than one configuration file, the process will exit with an error.
 
+By default, a node will not expose any RPC methods.
```

Comment:
Is disabling all APIs a good default? I figured it was best for the default to be maximally safe and to force the node operator to consider what APIs they want to enable, but I can see arguments the other way.